### ISSUE TITLE
Test times out on finding element but the screenshot shows the test in a completed state

### ISSUE THREAD

**Issue by colzaa** [SeleniumHQ/selenium#3152]
## Meta -
OS:  Ubuntu 16.04

Selenium Version:  3.0.0 

Browser:  Chrome

Browser Version:  Chrome v 54.0.2840.100 (64 bit)

Chromedriver Version: 2.25

## Expected Behavior -

When running tests I have a scenario which ends in 

And I press "Submit"
Then I should see "blah"

I expect that it submits the form via the submit button and the next page to be loaded is the show page of the item and that it contains the text "blah".

Any failures create a screenshot of the browser at the time of failure.

## Actual Behavior -
An exception is thrown at the stage " And I press "Submit" " stating that the element Submit could not be found.

The screenshot displays the show page for the item and that it contains the text "blah". 

The only way this page can be rendered is if the submit button was pressed.

In summary, the test is failing because it says it cannot find the button to click but it is also clicking the button and completing the test scenario.

**Comment by AutomatedTester** [comment]
For issues please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing and what results you expect.

See [CONTRIBUTING.md](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#issue-contributions)
